### DISCUSSION TITLE
Unable to get permission for SharePoint

### DISCUSSION THREAD

**Discussion by ritititi** [silverhack/monkey365#40] [Q&A]
Hi, 

I am scanning my tenant for Microsoft365 services, the scanner account has following roles:

SharePoint Administrator
Global Reader
Security Reader

I scanned all the M365 services, but the report is only generated for Exchange Online and, Security and Compliance.

There are no errors but the warning shows:
[Get-MonkeyPSPermission] - Unable to get permissions for https://mysharepointsite - warning - DesktopName - GetPsPSPermission

Are there any other settings I need to configure even though the user is SharePoint Admin?

**Comment by silverhack** [comment]
Hi @ritititi, 

Probably your user does not have enough permissions to access sites/Lists, etc.. Please, see docs regarding SharePoint site privileges:

[https://support.microsoft.com/en-us/office/overview-site-governance-permission-and-sharing-for-site-owners-95e83c3d-e1b0-4aae-9d08-e94dcaa4942e](https://support.microsoft.com/en-us/office/overview-site-governance-permission-and-sharing-for-site-owners-95e83c3d-e1b0-4aae-9d08-e94dcaa4942e)

Hope it helps,
Cheers

**Reply by ritititi** [reply]
Thank you for the reply.

I tried scanning on two tenants to see the difference. Both the scanner user has same roles:

SharePoint Administrator
Global Reader
Security Reader

Both of the tenant has same warning but only one tenant generates the sharepoint "Issues by service" report. Both of the user do not have permission to access all the sites and the warnings are normal. However, I cannot generate the overall "Issues by service" report for sharepoint on only one tenant. Could there be an additional configuration? Apart from the site access permission. 

Command used for both Tenant A and B
![image](https://user-images.githubusercontent.com/120542189/207792786-34881506-915d-42f7-ae0b-0a5e9402e6ff.png)

Tenant A
![image](https://user-images.githubusercontent.com/120542189/207792348-26634702-639e-4626-9126-3728c5762afd.png)

Tenant B
![image](https://user-images.githubusercontent.com/120542189/207792688-0e558ee8-5c52-4bb5-aaf6-f646fbbc447d.png)

Thank you for the help.

**Comment by silverhack** [comment]
Hi @ritititi, 

As previously mentioned, most cases are related to permission issues. Please, see docs regarding SharePoint site privileges, as well as difference between Edit and Contribute permissions:

[https://support.microsoft.com/en-us/office/overview-site-governance-permission-and-sharing-for-site-owners-95e83c3d-e1b0-4aae-9d08-e94dcaa4942e](https://support.microsoft.com/en-us/office/overview-site-governance-permission-and-sharing-for-site-owners-95e83c3d-e1b0-4aae-9d08-e94dcaa4942e)

[https://sharepointmaven.com/edit-vs-contribute-permission-levels-in-sharepoint-online/](https://sharepointmaven.com/edit-vs-contribute-permission-levels-in-sharepoint-online/)

Hope that helps.
Cheers,
